MachObfuscator/DependencyAnalysis/ObfuscableFilesFilter.swift

```diff
@@ -32,13 +47,17 @@ extension ObfuscableFilesFilter {
         }
     }
 
-    static func skipFramework(framework: String) -> ObfuscableFilesFilter {
+    static func isFramework(framework: String) -> ObfuscableFilesFilter {
         let frameworkComponent = framework + ".framework"
         return ObfuscableFilesFilter { url in
-            !url.pathComponents.contains(frameworkComponent)
+            url.pathComponents.contains(frameworkComponent)
         }
     }
 
+    static func skipFramework(framework: String) -> ObfuscableFilesFilter {
```

Comment:
What do you think about renaming this to make things more consistent? `skipFramework` -> `notFramework`

Reply:
IMO in `ObfuscableFilesFilter` and in option names `skip`/`obfuscate` are readable. What would be `not` prefix consistent with?

Reply:
`isFramework`, `none` describe characteristics. `skipFramework` seems to be a destructive operation. But I will not insist :)